core/vtk/ttkMergeTreeDistanceMatrix/ttk.module

```diff
@@ -0,0 +1,12 @@
+NAME
+  ttkMergeTreeDistanceMatrix
+SOURCES
+  ttkMergeTreeDistanceMatrix.cpp
+HEADERS
+  ttkMergeTreeDistanceMatrix.h
+DEPENDS
+  mergeTreeDistanceMatrix
+  ftmTree
```

Comment:
do you really need the dep to `ftmTree`?
(I guess `mergeTreeDistanceMatrix` may need it but not necessarily `ttkMergeTreeDistanceMatrix`)

Reply:
Indeed ! The dependency to ftmTree id removed and also ttkMergeTreeClustering.